source/deployment/bulk-loading-data-format.rst

```diff
@@ -773,6 +773,15 @@ Fields of the User object
           <kbd>"False"</kbd> </td>
       <td align="center" valign="middle">Yes</td>
       <td align="center" valign="middle">Yes</td>
+     <tr class="row-odd">
+      <td valign="middle">email_interval</td>
+      <td valign="middle">string</td>
+      <td>Specify an email batching interval during bulk import. Can have one of the following values:<br>
+          <kbd>"immediate"</kbd> - Emails are sent immediately.  <br>
+          <kbd>"fifteen"</kbd> - Emails are batched and sent every 15 minutes.<br>
+          <kbd>"hour"</kbd> - Emails are batched and sent every hour.<br> </td>
+      <td align="center" valign="middle">Yes</td>
+      <td align="center" valign="middle">Yes</td>
```

Comment:
Here is missing a `</tr>`, like in other groups before. It is not valid html, but probably the browsers are accepting it.

Reply:
Fixed.  Thank you!